wagtail/wagtailsnippets/tests.py

```diff
@@ -9,7 +9,8 @@
 
 from wagtail.tests.utils import WagtailTestUtils
 from wagtail.tests.testapp.models import Advert, SnippetChooserModel
-from wagtail.tests.snippets.models import AlphaSnippet, ZuluSnippet, RegisterDecorator, RegisterFunction, SearchableSnippet
+from wagtail.tests.snippets.models \
```

Comment:
The way we usually break an import like this to multiple lines is to use brackets:

```python
from wagtail.tests.snippets.models import (
    AlphaSnippet, ZuluSnippet, RegisterDecorator, RegisterFunction, SearchableSnippet
)
```